src/app/App.js

```diff
@@ -19,7 +19,7 @@ const GridArea = styled.div`
   display: grid;
```

Comment:
2nd grid is not necessary in my eyes because you dont use it as `grid`. you only set `padding` and `overflow`.
instead you can put these in your "mainpart"  `IdeasFeed` and `IdeaForm`